src/integ_test_resources/ios/sdk/integration/cdk/cdk_integration_tests_ios/iot_stack.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, scope: core.Construct, id: str, common_stack: CommonStack, **
         self.setup_iot_endpoint_provider()
         self.setup_custom_authorizer()
 
+        self.setup_custom_authorizerUserPass()
```

Comment:
I would `stick_with_snake()`

Reply:
For now, I am going to assume you are only referring to snake case for executing python code, and the strings we are using are fine to use camel?
 For example, this is okay?
```
        custom_authorizer_name = "iot_custom_authorizerUserPass"
```

Reply:
I don't favor mixed case even for keys--it's just one more thing to try to keep track of. We use snake case for all the configuration keys & strings except for the ones defined by `awsconfiguration.json` (and someday, `amplifyconfiguration.json`), so let's stick with snake case.

Reply:
Ok. That's fine.